### PR TITLE
Update Multiome.changelog.md for Cell Bender

### PR DESCRIPTION
**Pull request by aawdeh**
Need to update changelog to reflect changes in the Cell Bender version used. 

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by nikellepetrillo — APPROVED** [review]

File: pipelines/skylab/multiome/Multiome.changelog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.1
+2023-12-12 (Date of Last Commit)
+* Downgraded Cell Bender from v3.0.1 to v3.0.0
```
Comment (kayleemathews) [inline pipelines/skylab/multiome/Multiome.changelog.md:3]: Should this say v0.3.1 to v0.3.0?

Comment (aawdeh) [reply pipelines/skylab/multiome/Multiome.changelog.md:3]: Yes I think so. This is the version were using now. https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.0/wdl/cellbender_remove_background.wdl

Comment (aawdeh) [reply pipelines/skylab/multiome/Multiome.changelog.md:3]: Oh yes! I see it now

Comment (aawdeh) [reply pipelines/skylab/multiome/Multiome.changelog.md]: Thanks for pointing that out!

**Review by jessicaway — APPROVED** [review]

**Pull request merged**